src/sound_distance.py

```diff
@@ -0,0 +1,115 @@
+from math import sqrt
+import yaml
+
+
+class SoundDistance(object):
```

Comment:
a pydoc would be nice... there is not obvious how this distance is defined. please comment.

Reply:
ASAP